src/components/bs5/promotionalPanel/promotionalPanel.stories.js

```diff
@@ -0,0 +1,32 @@
+// modal.stories.js
+import { PromotionalPanel } from './promotionalPanel.js';
+import defaultdata from './promotionalPanel.data.json';
+
+export default {
+    tags: ["autodocs"],
+    title: "3. Components/Promotional Panel",
```

Comment:
maps to https://github.com/qld-gov-au/qgds-vanilla/blob/main/src/components/promo_panel/stories/promoPanel.stories.js#L17C24-L17C41 :+1: